docs/tutorials/hostport.md

```diff
@@ -198,3 +198,24 @@ kafka-1.ksvc.example.org
 kafka-2.ksvc.example.org
 ```
 
+#### Using node external IPs as targets
+
+Add the following annotation to your `Service`:
+
+```yaml
+external-dns.alpha.kubernetes.io/access: public
```

Comment:
Is this the right annotation to use? I am not experienced with headless records, this is the only entry that we have in the FAQ: https://github.com/kubernetes-sigs/external-dns/blob/cc9a60ecfa7981d7cbf7cf853f2107d4bfc1c2e8/docs/faq.md#L277 . 

Can you motivate why you reused that annotation?

Reply:
Yes, I would prefer to use a new annotation to avoid overloading meanings of previously used annotation. Thanks!

Reply:
Changed to `external-dns.alpha.kubernetes.io/endpoints-type: HostIP|NodeExternalIP`